lib/assortment.dart

```diff
@@ -94,7 +94,6 @@ class Assortment {
       _dragging = true;
       // use the drag icon for moving
       event.dataTransfer.effectAllowed = "move";
-      event.dataTransfer.setData('text/plain', '');
```

Comment:
This line causes IE drag and drop to fail. 
event.dataTransfer.setData('text'); could be used which is accept across Chrome, firefox and IE 11. But since we are not using this setData, it can be completely removed and the functionality is not affected.